client/lib/plans/features-list.js

```diff
@@ -510,7 +510,7 @@ export const FEATURES_LIST = {
 
 	[ constants.FEATURE_UPLOAD_PLUGINS ]: {
 		getSlug: () => constants.FEATURE_UPLOAD_PLUGINS,
-		getTitle: () => i18n.translate( 'Install Plugins' ),
+		getTitle: () => i18n.translate( 'Install plugins' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.